subprojects/language-native/src/integTest/groovy/org/gradle/language/nativeplatform/ParallelNativePluginsIntegrationTest.groovy

```diff
@@ -41,6 +43,7 @@ class ParallelNativePluginsIntegrationTest extends AbstractInstalledToolChainInt
     }
 
     @Requires(TestPrecondition.OBJECTIVE_C_SUPPORT)
```

Comment:
We've now got 2 ways of doing the same thing. We should either replace all usages of `TestPrecondition.OBJECTIVE_C_SUPPORT` with the new annotation, _or_ replace the new annotation with some new `TestPrecondition` cases.